src/lib/Containers/Conversation.tsx

```diff
@@ -186,6 +203,40 @@ export class Conversation extends React.Component<Props, State> {
   }
 }
 
+function markLastMessageRead(
```

Comment:
FYI I moved mutations to their own module to keep them separate from the component and keep the module at reasonable size in my PR
https://github.com/artsy/emission/pull/750/files#diff-a5d0421b6561ab82cfebdd123faadb42

Reply:
Cool, I'll do that here since this file is getting pretty big, good idea!

Also, that should hopefully minimize conflicts for whomever.